actioncable/CHANGELOG.md

```diff
@@ -1,3 +1,42 @@
+*   The ActionCable javascript package has been converted from CoffeeScript
+    to ES2015, and we now publish the source code in the npm distribution.
+
+    This allows ActionCable users to depend on the javascript source code
+    rather than the compiled code, which can produce smaller javascript bundles.
+
+    This change includes some breaking changes to optional parts of the
+    ActionCable javascript API:
```

Comment:
This change also means applications using the npm package need to
```diff
-import ActionCable from 'actioncable';
+import * as ActionCable from 'actioncable';
```
right? If so, that seems like the biggest "breaking" change to call out.

Reply:
Actually that's not necessary. Both will work, due to how rollup generates the bundle. The later is only necessary if you import the untranspiled source code (which wasn't an option before this PR, so it's not a breaking change).

Reply:
The generator template will need updating too: https://github.com/rails/rails/blob/a429b29425027006f2bbd4d267928dde1dc2c31a/railties/lib/rails/generators/rails/app/templates/app/javascript/channels/consumer.js#L1-L6

Reply:
> Actually that's not necessary.

Great :)